ebuild-maintenance/package-and-slot-moves/text.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0"?>
+<guide self="ebuild-maintenance/maintenance-tasks/">
+<chapter>
+<title>Package and slot moves</title>
+
+<body>
+<p>
+This chapter describes the use of package and slot moves. The package update
+mechanism is a powerful too, and needs to be used with caution. In particular,
+the following must be noted:
+</p>
+
+<ol>
+  <li>
+    Updates are <e>not one-shot</e> operations and they are not stateful. All
+    updates can be reapplied multiple times to the same system, and all old
+    updates are applied to fresh Gentoo installations.
+  </li>
+
+  <li>
+    Once an update entry is created, the old package name (slot) cannot be
+    reused. Attempting to reuse it will cause updates to apply again,
+    to the reused name. This also means that updates cannot be undone.
+  </li>
+
+  <li>
+    Updates can only perform 1:1 moves. They cannot be used to merge packages.
+    Attempting to move two or more packages into a single name may cause serious
+    problems for the users.
+  </li>
+</ol>
+</body>
+
+<section>
+<title>Moving or renaming a package</title>
+<body>
+
+<p>
+Moving or renaming a package requires several operations. Firstly, verify that
+the ebuilds will continue to work correctly after the move. If the category
+changes, you must verify all <c>${CATEGORY}</c> uses. If the package name
+changes, you must verify <c>${PN}</c>, <c>${P}</c>, etc. Whenever the old value
+is necessary, inline it. Commit the changes separately before the move.
```

Comment:
What do  you mean by inlining here? Substituting the variable that represents the old package cat/name with the verbatim text, so that it doesn't get affected by the move? Also what's the rationale for creating a separate commit for those before the move?

Reply:
Yes. The rationale is to clearly split the move into smaller changes, and make the diff more readable. Doing a move in one commit is like using a merge commit to alter some files.